test/04_build_skip/setup.py

```diff
@@ -1,13 +1,6 @@
 from setuptools import setup, Extension
 import sys
 
-if sys.argv[-1] != '--name':
```

Comment:
Same here. We dó want these tests, but since you've removed the call with the `--name` argument, the condition and if-statement can go, keeping the body.